src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,127 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTORY = 0.75f;
```

Comment:
```suggestion
    private static final float LOAD_FACTOR = 0.75f;
```
Our load factor will be 0.75 no matter what